packages/template-retail-react-app/app/commerce-api/index.js

```diff
@@ -58,7 +59,16 @@ class CommerceAPI {
 
         this._config = {proxy, ...restConfig}
 
-        this.auth = new Auth(this)
+        this._authConfig = {
+            redirectURI: `${getAppOrigin()}/callback`,
+            proxy,
+            locale: this._config.locale,
+            currency: this._config.currency,
+            ...this._config.parameters,
+            ...this._config.headers
+        }
```

Comment:
Would you consider creating a common configuration that we can reuse for both the commerce-api and `commerce-sdk-react`?

Reply:
Yeah that's a part of the refactoring `willSendRequest` and SDK initialization ticket.